client/components/woopay/save-user/test/checkout-page-save-user.test.js

```diff
@@ -173,18 +188,9 @@ describe( 'CheckoutPageSaveUser', () => {
 	} );
 
 	it( 'call `extensionCartUpdate` on blocks checkout when checkbox is clicked', async () => {
-		extensionCartUpdate.mockResolvedValue( {} );
```

Comment:
Refactored to use a wrapper that emulates the blocks checkout, since the same setup is used in multiple tests (see `BlocksCheckoutEnvironmentMock` above)